api/proto/resourcespb/database.proto

```diff
@@ -53,8 +54,9 @@ message DatabaseResource {
   string username = 7;
   string password = 8;
   repeated string subnet_ids = 9;
+  int32 port = 10;
```

Comment:
Just use 12 here.

In protobufs, if you change the number of existing values (host and connection_username), you lose backwards compability. This explains better how protbuf encoding works: https://developers.google.com/protocol-buffers/docs/encoding#structure